tests/api.spec.ts

```diff
@@ -16,10 +16,23 @@ import {
   reportEvent,
 } from '../src/http';
 
-const fakeBundleId = '769e1811db5e98abdb65df9160228ad51646f535af28b80a8913813aec1bd331';
+const fakeBundleId = 'e87a6db9b99f0f91a4ce6d0894eff3f6be7d5c49d9849cbd0258b96d3fe58378';
 let fakeBundleIdFull = '';
-const realBundleId = 'ffdd7b1cb8e28d3859e6b7179bf4b871d1abd8a7a2cacf0002d5174aca3b6841';
+const realBundleId = '1d16a90ae18e96edf2ab21e442a7c0665939613e4a2a9831f98c7cf32bcd5e00';
 let realBundleIdFull = '';
+const extendedBundleId = '587a6bcb0095606ad57ccc7bb7ac6401475ce4181c13f7136491a16df06544f1';
+
+const fakeMissingFiles = [
+  `/GitHubAccessTokenScrambler12.java`,
+  `/app.js`,
+  `/db.js`,
+  `/main.js`,
+  // TODO: This should be ignored
```

Comment:
Hard to do since we use the same sample repo to do all the folder tests. An option would be to have multiple sample repos each one for a given test case but I think it would be hard to maintain.